examples/seismic/acoustic/operators.py

```diff
@@ -107,8 +109,60 @@ def iso_stencil(field, model, kernel, **kwargs):
     return eqns
 
 
+def create_snapshot_time_function(model, name, geometry, space_order, factor=None):
```

Comment:
I would lift that into `seismic/utils.py` or some new file so it can be easily reused from other pdes. It can probably be done directly from the wavefield too i,e

```
def create_snapshot_time_function(u, nsnap):

    name = f"{u.name}_save"
    grid = u.grid
    
    ....
    
 ```

Makes it also decoupled from those geometry/model. THis would also allow to directly return the equation `Eq(u,usave)` as well